### PR TITLE
Suisin/wall2278/fix code review

### PR DESCRIPTION
**Pull request by suisin-deriv**
## Changes:

Fix commented code in the PR

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [X] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/bot-web-ui/src/components/dashboard/bot-builder/bot-builder.tsx [hunk 1]
```diff
@@ -26,9 +26,13 @@ const BotBuilder = observer(() => {
...
 
     const handleBlockChangeOnBotRun = (e: Event) => {
-        if (e.type !== 'ui') {
+        const { is_reset_button_clicked, setResetButtonState } = toolbar;
+        if (e.type !== 'ui' && !is_reset_button_clicked) {
             setShowSnackbar(true);
             removeBlockChangeListener();
+        } else if (is_reset_button_clicked) {
+            setResetButtonState(false);
+            removeBlockChangeListener();
```
Comment (amina-deriv) [inline packages/bot-web-ui/src/components/dashboard/bot-builder/bot-builder.tsx:35]: This is not related to this PR, Lets revert this file.

File: packages/bot-web-ui/src/components/dashboard/bot-builder/toolbar/toolbar.tsx [hunk 2]
```diff
@@ -69,7 +69,7 @@ const Toolbar = observer(() => {
             >
                 {is_running ? (
                     <Localize
-                        i18n_default_text='Deriv Bot will not proceed with any new trades. Any ongoing trades will be completed by our system. Any unsaved changes will be lost.<0>Note: Please check your statement to view completed transactions.</0>'
+                        i18n_default_text='The workspace will be reset to the default strategy and any unsaved changes will be lost. <0>Note: This will not affect your running bot.</0>'
                         components={[
```
Comment (amina-deriv) [inline packages/bot-web-ui/src/components/dashboard/bot-builder/toolbar/toolbar.tsx:73]: Lets revert this file change as it is unrelated

File: packages/core/src/Stores/ui-store.js [hunk 3]
```diff
@@ -89,7 +89,7 @@ export default class UIStore extends BaseStore {
     has_real_account_signup_ended = false;
 
     // verification modal
-    is_verification_modal_visible = false;
+    is_verification_modal_visible = true;
```
Comment (amina-deriv) [inline packages/core/src/Stores/ui-store.js:92]: why do we set default value as true?

Comment (suisin-deriv) [reply packages/core/src/Stores/ui-store.js:92]: oops was testing the modal 😓 let me change this back 🙏

File: packages/appstore/src/components/wallet-header/wallet-header.tsx [hunk 4]
```diff
@@ -16,10 +16,9 @@ type TWalletHeader = {
 
 const WalletHeader = observer(({ wallet_account }: TWalletHeader) => {
     const { client, traders_hub } = useStore();
```
Comment (amina-deriv) [inline packages/appstore/src/components/wallet-header/wallet-header.tsx:18]: Is traders_hub used anywhere in this file

Comment (suisin-deriv) [reply packages/appstore/src/components/wallet-header/wallet-header.tsx:18]: will remove it

**Review by amina-deriv — APPROVED** [review]
